fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/query/plan/plans/RecordQueryCoveringIndexPlan.java

```diff
@@ -193,4 +195,14 @@ public int planHash() {
     public List<? extends Quantifier> getQuantifiers() {
         return ImmutableList.of();
     }
+
+    @Nonnull
+    @Override
+    public PlannerGraph rewritePlannerGraph(@Nonnull final List<? extends PlannerGraph> childGraphs) {
+        final PlannerGraph indexPlannerGraph = indexPlan.rewritePlannerGraph(childGraphs);
+        return PlannerGraph.fromNodeAndChildGraphs(
+                new PlannerGraph.OperatorNodeWithInfo(this,
+                        NodeInfo.COVERING_INDEX_SCAN_OPERATOR),
```

Comment:
I think that the covering index plan should basically look like an index plan, but with a different operator name, rather than having  regular index plan as a child, even though that's how it's implemented.

Reply:
Done.